js/ui/status/keyboard.js

```diff
@@ -488,6 +497,45 @@ var InputSourceManager = class {
             this._updateMruSettings();
     }
 
+    get PasswordModeEnabled() {
+        return this.passwordModeEnabled;
```

Comment:
This look strange to me, why do we have 2 getters and setters for this property? And why one of them uses `this._passwordModeEnabled` while the other uses `this.passwordModeEnabled`? Same is valid for the setter below.

Reply:
One is the D-Bus property implementation, the other is the JavaScript property.

Reply:
I see, thanks.